lib/util/git/index.ts

```diff
@@ -784,30 +805,14 @@ export async function commitFiles({
       return null;
     }
 
-    const pushOptions: TaskOptions = {
-      '--force-with-lease': null,
-      '-u': null,
-    };
-    if (getNoVerify().includes(GitNoVerifyOption.Push)) {
-      pushOptions['--no-verify'] = null;
-    }
-
-    const pushRes = await git.push(
-      'origin',
-      `${branchName}:${branchName}`,
-      pushOptions
-    );
-    delete pushRes.repo;
-    logger.debug({ result: pushRes }, 'git push');
-    // Fetch it after create
-    const ref = `refs/heads/${branchName}:refs/remotes/origin/${branchName}`;
-    await git.fetch(['origin', ref, '--force']);
-    config.branchCommits[branchName] = (
-      await git.revparse([branchName])
-    ).trim();
+    const commit = await pushCallback({
+      ...commitConfig,
+      files: files.filter(({ name }) => !ignoredFiles.includes(name)),
+    });
+    config.branchCommits[branchName] = commit;
     config.branchIsModified[branchName] = false;
     incLimitedValue(Limit.Commits);
-    return commit;
+    return commit ? commit.slice(0, 7) : 'unknown';
```

Comment:
```suggestion
    return commit?.slice(0, 7) ?? 'unknown';
```
are we sure to return the short id?

Reply:
Seems like it really isn't being used anywhere